charts/tidb-backup/values.yaml

```diff
@@ -94,6 +106,7 @@ s3: {}
   # region: ""
   # bucket: ""
   # secretName is the name of the secret which stores s3 object store access key and secret key
+  # This is not necessary on AWS. Instead you should be able to get the credentials from the EKS service IAM role.
```

Comment:
Then how to use it in the job container?

Reply:
The AWS SDK will find the credentials, it just needs to run with a service account. You have to be using EKS or properly running something like iam-authenticator in the k8s cluster.

Reply:
We just need to attach the service account, no need to install the SDK or call any command to run with it?

Reply:
Does iam-authenticator need to be packaged in the Docker image?

Reply:
yes, these changes are tested as working. Just set the service account.
On GCP the association with the k8s service account to the cloud service account is done with

```
kubectl -n operations annotate serviceaccount ${svcacct} iam.gke.io/gcp-service-account=${svcacct}@${PROJECT}.iam.gserviceaccount.com
```

On EKS I believe one just needs to attach a policy. If not using EKS you run an authenticator program for the entire cluster.  https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html